frontend/src/components/Modal/RiderModalInfo.tsx

```diff
@@ -131,22 +132,15 @@ const RiderModalInfo = ({
           )}
         </div>
         <div className={cn(styles.gridR2, styles.gridCBig1)}>
-          <Label className={styles.label} htmlFor="needs">
-            Needs:{' '}
-          </Label>
-          <select
-            name="needs"
-            aria-required="true"
-            ref={register({ required: true })}
-          >
-            {Object.values(Accessibility).map((value, index) => {
-              return (
-                <option key={index} value={value}>
-                  {value}
-                </option>
-              );
-            })}
-          </select>
+          Needs:
```

Comment:
The reason why this is not bolded when the screen size is changed is because it is not wrapped using a label. 

The headers for each input type uses a label and there is a media-query which keeps the label bolded on different screen sizes. Check out the code in `src/components/modal/ridermodal.module.css`

Needs should be wrapped like the other labels 
```
          <Label className={styles.label} htmlFor="needs">
            Needs:{' '}
          </Label>
```